cypress/integration/core-opensearch-dashboards/opensearch-dashboards/apps/vis_type_table/basic.spec.js

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  TABLE_INDEX_ID,
+  TABLE_INDEX_PATTERN,
+  TABLE_PATH_INDEX_DATA,
+  TABLE_INDEX_START_TIME,
+  TABLE_INDEX_END_TIME,
+  TABLE_CREATE_URL,
+  TABLE_VIS_APP_PATH,
+  TABLE_VIS_TYPE,
+} from '../../../../../utils/constants';
+
+/**
+ * Basic test suite description:
+ * Create a new table visualization from index pattern.
+ * Set the time range and Histogram Aggregation.
+ * Test that global header and side bar function correctly.
+ * Test that table visualization can be saved.
+ * Test inspector is enabled and contains correct data.
+ */
+
+describe('table visualization basic functions', () => {
+  before(() => {
+    cy.deleteIndex(TABLE_INDEX_ID);
+    cy.deleteIndexPattern(TABLE_INDEX_PATTERN);
+    cy.bulkUploadDocs(TABLE_PATH_INDEX_DATA);
+    cy.createIndexPattern(TABLE_INDEX_PATTERN, {
```

Comment:
your saved object seems to already have the required index pattern. Do you still need to run this command?

Reply:
Yes. For saved index pattern, I used two ways to test in table. 
1) Manually add an index pattern in saved object then create a table vis `cy.createIndexPattern`. This is used in basic.spec.js
2) Import an saved object through `cy.importSavedObjects(TABLE_PATH_SO_DATA)`. This method is used in other three tests. 
Basically I would like to try both to make sure table vis can created from both ways.